src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
+  } else if (days >= 3) {
+    result = result - 20;
+  }
```

Comment:
According to the checklist:  _use variables for the main values so that you can reuse them and give them descriptive names._ you should creat proper variables to store: price and discrounts